translator-files/po/de/Installation-for-Macintosh.po

```diff
@@ -15,6 +15,8 @@ msgstr ""
 "Content-Transfer-Encoding: 8bit\n"
 "Plural-Forms: nplurals=2; plural=(n != 1);\n"
 "X-Generator: Lokalize 19.12.3\n"
+"X-Language: de_DE\n"
+"X-Source-Language: C\n"
```

Comment:
Added by Qt creator.

Reply:
Yes, some editors add metadata but it's not a problem.